db/models/aggregate_functions.py

```diff
@@ -185,7 +185,11 @@ def update_application_and_related_form(
     form_sql_row = FormsMethods.get_form(application_id, form_name)
 
     if form_name == "project-information":
-        fields_array = question_json[1]["fields"]
+        current_app.logger.error(question_json)
```

Comment:
I guess this was left in for debugging, can we remove it?

Reply:
oh yes! sorry, will get rid of that 👍🏾